.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,14 +2,37 @@ refs #
 
 <!-- if your contribution is a new calendar -->
 
-For information, read and make sure you're okay with the [CONTRIBUTING document](https://github.com/novafloss/workalendar/blob/master/CONTRIBUTING.rst#adding-new-calendars).
+For information, read and make sure you're okay with the [Contributing guidelines](https://github.com/novafloss/workalendar/blob/master/contributing.md#adding-new-calendars).
```

Comment:
Probably this document should be customized for the Calendra fork, probably pointing to its own contributing doc. Perfectly fair to leave this out of scope for now.